.pre-commit-config.yaml

```diff
@@ -4,9 +4,10 @@
 
 repos:
 -   repo: https://github.com/python/black
-    rev: 20.8b1
+    rev: 22.1.0
```

Comment:
`Even newer version available; we are using it in CPy libs:
``suggestion
    rev: 22.3.0
```

Reply:
Updated